### ISSUE TITLE
Feature request: Github Commit/branch Status 

### ISSUE THREAD

**Issue by tunnckoCore** [amio/badgen-service#157]
https://developer.github.com/v3/repos/statuses/

```
badgen.net/github/status/babel/babel
badgen.net/github/status/babel/babel/[branch | commit | ref]
badgen.net/github/status/babel/babel/foobar
badgen.net/github/status/babel/babel/35h5jkh56k
```

All this will play well with the new Now + Github enhancement. You will just point to the github status to have "build status badge".

**Comment by tunnckoCore** [comment]
So

https://badgen.net/github/status/tunnckoCore/foobar?label=build&icon=github
![](https://badgen.net/badge/status/pending/orange?label=build&icon=github)

https://badgen.net/github/status/tunnckoCore/foobar/master
![](https://badgen.net/badge/status/success/green)

https://badgen.net/github/status/tunnckoCore/foobar/failing-branch?label=build
![](https://badgen.net/badge/build/failure/red)

https://badgen.net/github/status/tunnckoCore/foobar/errrrr
![](https://badgen.net/badge/status/error/red)

**Comment by amio** [comment]
This is resolved by #161 
Thanks @tunnckoCore

**Comment by tunnckoCore** [comment]
Great! : )
